_life/money-americans.md

```diff
@@ -76,6 +76,10 @@ particular (as opposed to other financial services that I don't know much about)
 large, international banks, like UBS and Credit Suisse, are generally better at this
 than the smaller, local ones.
 
+Note that Swiss bank accounts typically have monthly maintenance fees that they are
+less likely to waive for you than in the US (where waivers for being a student, having
```

Comment:
Do the student accounts not work for PhD students? e.g., https://www.bcv.ch/fr/home/jeunes/produits/gestion-courante.html or https://www.ubs.com/ch/en/private/accounts-and-cards/bundles/students.html

Reply:
they very well might, I’m not ruling it out, but UBS was unwilling to give me the waiver (but there was also some miscommunication about whether I’m a PhD “student” or an employee)

Reply:
ah, good old "PhD students are whichever of student or employee is the least convenient at any given moment" :-)